docs/repairs-api/endpoints/tenant-leasehold-property-repair-cancel.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 6
+---
+
+# Tenant Or Leasehold Property Repair Cancel
+
+## GET: `/TenantOrLeaseholdPropertyRepairCancel`
+
+### Request:
+
+#### Authorization
+
+Bearer Token `<token>`
+
+#### Query Params
+
+| Key            | Example value  | Required |
+|----------------|----------------| -------- |
+| postcode       | "M3 0W"        | ✅       |
+| repairId       | "9C36D811"     | ✅       |
+
+### Response:
+
+HTTP 200 - success
```

Comment:
Nitpick: use consistent casing for status text, i.e. either all lower case, sentence casing or Pascal case (with spaces), not a combination.